test/test_sorted_containers.jl

```diff
@@ -1300,10 +1297,6 @@ end
     factors5 = SortedMultiDict(test_pair_array)
     @test typeof(factors5) == SortedMultiDict{Char,Any,ForwardOrdering}
 
-    println("The following warning is expected:")
-    factors6 = SortedMultiDict(['a', 'b'], [1, 2])
-    @test typeof(factors6) == SortedMultiDict{Char,Int,ForwardOrdering}
-
     #@test factors2 == factors3   # Broken!  TODO: fix me...
```

Comment:
This line looks like it should have been deleted long ago,
or changed to an `@test_broken`

Not related to this PR though